diff --git a/lib/mempcpy.c b/lib/mempcpy.c
index d700cc1..5b58ae3 100644
--- a/lib/mempcpy.c
+++ b/lib/mempcpy.c
@@ -20,8 +20,6 @@
 #include <string.h>
 
 /* A function definition is only needed if HAVE_MEMPCPY is not defined.  */
-#if !HAVE_MEMPCPY
-
 /* Copy N bytes of SRC to DEST, return pointer to bytes after the
    last written byte.  */
 void *
@@ -29,5 +27,3 @@ mempcpy (void *dest, const void *src, size_t n)
 {
   return (char *) memcpy (dest, src, n) + n;
 }
-
-#endif
diff --git a/lib/string.in.h b/lib/string.in.h
index c76c182..5966dff 100644
--- a/lib/string.in.h
+++ b/lib/string.in.h
@@ -230,25 +230,8 @@ _GL_WARN_ON_USE (memmem, "memmem is unportable and often quadratic - "
 
 /* Copy N bytes of SRC to DEST, return pointer to bytes after the
    last written byte.  */
-#if @GNULIB_MEMPCPY@
-# if ! @HAVE_MEMPCPY@
-_GL_FUNCDECL_SYS (mempcpy, void *,
-                  (void *restrict __dest, void const *restrict __src,
-                   size_t __n)
-                  _GL_ARG_NONNULL ((1, 2)));
-# endif
-_GL_CXXALIAS_SYS (mempcpy, void *,
-                  (void *restrict __dest, void const *restrict __src,
-                   size_t __n));
-_GL_CXXALIASWARN (mempcpy);
-#elif defined GNULIB_POSIXCHECK
-# undef mempcpy
-# if HAVE_RAW_DECL_MEMPCPY
-_GL_WARN_ON_USE (mempcpy, "mempcpy is unportable - "
-                 "use gnulib module mempcpy for portability");
-# endif
-#endif
 
+void* mempcpy (void *dest, const void *src, size_t n);
 /* Search backwards through a block for a byte (specified as an int).  */
 #if @GNULIB_MEMRCHR@
 # if ! @HAVE_DECL_MEMRCHR@
