diff --git a/miscutils/time.c b/miscutils/time.c
index c4bcbcb..0006c59 100644
--- a/miscutils/time.c
+++ b/miscutils/time.c
@@ -33,13 +33,6 @@
 
 #include "libbb.h"
 
-#ifndef HAVE_WAIT3
-static pid_t wait3(int *status, int options, struct rusage *rusage)
-{
-	return wait4(-1, status, options, rusage);
-}
-#endif
-
 /* Information on the resources used by a child process.  */
 typedef struct {
 	int waitstatus;
